roles/static_hostname_lookup/templates/hosts.j2

```diff
@@ -42,6 +42,15 @@
 {% endif %}
 {% if groups['sys_admin_interface'] is defined and groups['sys_admin_interface'] | length >= 1 %}
```

Comment:
This line here:
```{% if groups['sys_admin_interface'] is defined and groups['sys_admin_interface'] | length >= 1 %}```
Needs to be
```{% if groups['irods'] is defined and groups['irods'] | length >= 1 %}```
here. Note that the next instance of
```{% if groups['sys_admin_interface'] is defined and groups['sys_admin_interface'] | length >= 1 %}```
on line 52, which is really for the DAIs and SAIs, should remain as is.